backend/inspirehep/records/indexer/cli.py

```diff
@@ -239,3 +240,56 @@ def _finished_tasks_count():
         )
         for failure in failures:
             LOGGER.warning(failure)
+
+
+@index.command(
+    "remap",
+    help="Remaps specified indexes. Removes all data from index during this process.",
+)
+@click.option("--yes-i-know", is_flag=True)
+@click.option(
+    "--index",
+    "-i",
+    multiple=True,
+    default=None,
+    help="Specify indexes which you want to remap (ignore prefix and postfix)",
+)
+@click.option(
+    "--ignore-checks",
+    is_flag=True,
+    help="Do not check if old index was deleted. WARNING: this may lead to one alias pointing to many indexes!",
+)
+@with_appcontext
+@click.pass_context
+def remap_indexes(ctx, yes_i_know, index, ignore_checks):
+    if not index:
+        click.echo("You should specify indexes which you want to remap")
+        ctx.exit(1)
+    for index_name in index:
```

Comment:
```python
current_indexes = current_search.mappings.keys()
valid_indexes =  set(current_indexes).intersection(indexes)
if not valid_indexes:
   click.echo ...
```

Reply:
I believe that we want to stop on any wrong index, not only if there will be no valid indexes.